static/app/views/settings/project/projectOwnership/codeowners.tsx

```diff
@@ -35,7 +35,7 @@ class CodeOwnersPanel extends Component<Props> {
       addErrorMessage(t('An error occurred'));
     }
   };
-
+  // handle sync
```

Comment:
Is this in here on purpose?

Reply:
nope! I just used it as a marker for relevant code.